6.3.0/aspnet-core/src/BhResturant.Core/Products/Product.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Abp.Domain.Entities;
+
+namespace BhResturant.Products
+{
+    public class Product : Entity<int>
+    {
+        public string Name { get; set; }
+        public int Price { get; set; }
+    }
+}
```

Comment:
again? seems weird you have to declrae the same dto 3 times in the application. should be 1